static/js/components/EmploymentTab.js

```diff
@@ -1,21 +1,131 @@
 import React from 'react';
+import Button from 'react-mdl/lib/Button';
+import Grid from 'react-mdl/lib/Grid';
+import { Cell } from 'react-mdl/lib/Grid';
+
+import ProfileTab from "../util/ProfileTab";
+import { saveAndContinue } from '../util/profile_edit';
+
+class EmploymentTab extends ProfileTab {
+  constructor(props) {
+    super(props);
+    this.saveAndContinue = saveAndContinue.bind(this, '/dashboard', this.validation);
+    this.blankWorkHistoryEntry = {
+      id: null,
+      position: "",
+      industry: "",
+      company_name: "",
+      start_date: "",
+      end_date: "",
+      city: "",
+      country: "",
```

Comment:
I feel like it's an oversight that it's not in the mockups. I'll see if I can grab @pdpinch to be sure